module-4/backend/week_one.md

```diff
@@ -5,15 +5,41 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+I really like learning about how front end and back end programs are completely seperate programs and connect through APIs.  Maybe that isn't as sexy as learning AJAX or something but to me this changes the way I look at programming and opens up a lot of ideas for my personal project (like a golang backend and a react frontend)
+
 2. What are some tools to help debug JavaScript code?
+I find debugging in JS much harder than Ruby.  The error codes are not as helpful!  I use debugger, the console and the error messages in the npm start webpack server compiler to help debug.  This is an area that I feel I could def improve on so I will keep working on it.  
+
 3. What are some tools you need in order to unit test your JavaScript?
+In class we used Mocha and Chai.  Mocha is for unit testing and Chai adds a bunch of useful methods like deepEqual that normally wouldn't say two strings or two arrays are equal because they are in different places in memory.
+
 4. What is the syntax for invoking a function?
+function multiply() {
+statements
+};
+let functionName = () = {
+statements
+};
+var multiply = new Function(x, y, statements);
+
```

Comment:
these are all ways to define a function. how do you invoke/call a function in JS?